source/css/elements/_video.scss

```
@@ -10,25 +10,28 @@
   @media (min-width: $screen-tablet) {
     border: 5px solid #ccc;
   }
-}
 
-.external_video {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  border: none;
+  iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border: none;
+  }
 }
 
+
 .video_overlay {
+  background: rgba(#000, 0.12); // if we don't have a background image
```

Is this SCSS syntax?

```suggestion
  background: rgba(0, 0, 0, 0.12); // if we don't have a background image
```

---

Yeah, SCSS.  Works, but I agree.